<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UMessageBoxClass</class>
 <widget class="QDialog" name="UMessageBoxClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>330</width>
    <height>132</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UMessageBoxClass</string>
  </property>
  <widget class="QWidget" name="shadowWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>330</width>
     <height>132</height>
    </rect>
   </property>
   <widget class="QWidget" name="centralWidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>330</width>
      <height>132</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>330</width>
      <height>132</height>
     </size>
    </property>
    <widget class="QLabel" name="messagebox_title">
     <property name="geometry">
      <rect>
       <x>41</x>
       <y>14</y>
       <width>101</width>
       <height>14</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei UI</family>
      </font>
     </property>
     <property name="text">
      <string>有盟装机助手</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="messagebox_ck">
     <property name="geometry">
      <rect>
       <x>23</x>
       <y>97</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei UI</family>
      </font>
     </property>
     <property name="text">
      <string>不再提示</string>
     </property>
    </widget>
    <widget class="QLabel" name="messagebox_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>56</y>
       <width>316</width>
       <height>14</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei UI</family>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="messagebox_confirm">
     <property name="geometry">
      <rect>
       <x>204</x>
       <y>91</y>
       <width>52</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei UI</family>
      </font>
     </property>
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
    <widget class="QPushButton" name="messagebox_cancel">
     <property name="geometry">
      <rect>
       <x>265</x>
       <y>91</y>
       <width>52</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft JhengHei UI</family>
      </font>
     </property>
     <property name="text">
      <string>取消</string>
     </property>
    </widget>
    <widget class="QPushButton" name="messagebox_close">
     <property name="geometry">
      <rect>
       <x>305</x>
       <y>14</y>
       <width>11</width>
       <height>11</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../UmInstallerAssistant.qrc"/>
 </resources>
 <connections/>
</ui>
